doc/manual/source/man/ldns-nsec3-hash.rst

```diff
@@ -0,0 +1,29 @@
+ldns-nsec3-hash
+===============
+
+Synopsis
+--------
+
+:program:`ldns-nsec3-hash` :samp:`<{domain-name}>`
```

Comment:
Somewhere else you changed the `:samp:` to just uppercase with backticks.

Reply:
Oh, I thought I changed that. Let me change that to again to backticks ^^ 
(we're not really using the variable feature offered by `:samp:` so I changed it to be consistent with the rest)